internal/pkg/manifest/env.go

```diff
@@ -114,3 +115,11 @@ type environmentObservability struct {
 func (o environmentObservability) IsEmpty() bool {
 	return o.ContainerInsights == nil
 }
+
+type environmentHTTPConfig struct {
+	Internet internetHTTPConfig `yaml:"internet"`
```

Comment:
I'm torn about the field name, because we have "public" subnets, `network.placement: 'public'`, and app runner also uses the term "public" for endpoints that have access from the internet (https://aws.amazon.com/blogs/containers/deep-dive-on-aws-app-runner-vpc-networking/). On the other hand, for elb it's `internet-facing`.

I'm leaning towards `public` instead what do you think? for consistency

Reply:
So it's `internet` & `internal` vs. `public` & `private` right?

I think what you proposed sound good. Both `network.public` & `http.public` would map to something "with a public IP", & `network.private` & `http.private` map to something without a public IP.

Reply:
yeah, I think that makes sense to me too